README.md

```diff
@@ -70,21 +70,20 @@ func main() {
     client, err := onepassword.NewClient(
                 context.TODO(),
                 onepassword.WithServiceAccountToken(token),
+				// TODO: Set the following to your own integration name and version.
```

Comment:
The ndentation seems to be incorrect here

Reply:
done